api/nakadi-event-bus-api.yaml

```diff
@@ -548,6 +548,42 @@ paths:
           description: |
             Schema object.
 
+  '/event-types/{name}/cursor-distances':
+    get:
+      tags:
+        - unmanaged-api
+        - monitoring
+        - management-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        Calculate the distance between two offsets. This is useful for performing checks for data completeness, when
+        a client has read some batches and wants to be sure that all delivered events have been correctly processed.
+      parameters:
```

Comment:
The `name` path parameter is missing here ... I guess this is just a single oversight?